### PR TITLE
Add item remove button and function

### PR DESCRIPTION
**Pull request by JanekJaniak**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -26,14 +25,19 @@ export default {
   },
   methods: {
     addItem(value) {
+      if(value == "") return;
```
Comment (jan-gajdowicz) [inline src/App.vue]: is the same as `if(!value)`

File: src/App.vue [hunk 2]
```diff
@@ -26,14 +25,19 @@ export default {
...
       this.items.push(value);
     },
+    removeItem(itemID) {
+      this.items = this.items.filter((item) => {
+        return item.id != itemID;
+      })
```
Comment (jan-gajdowicz) [inline src/App.vue]: `this.items = this.items.filter(item => item.id != itemID)`

File: src/components/ItemCreator.vue [hunk 3]
```diff
@@ -22,6 +22,7 @@ export default {
         value: this.value,
         id: Date.now(),
       };
+      if(this.value== "") return;
```
Comment (jan-gajdowicz) [inline src/components/ItemCreator.vue]: `if(!this.value) return`

File: src/components/ItemCreator.vue [hunk 4]
```diff
@@ -22,6 +22,7 @@ export default {
...
       this.$emit("create", item);
       this.value= "";
```
Comment (jan-gajdowicz) [inline src/components/ItemCreator.vue]: `this.value = ""` whitespace!

File: src/components/ItemRemover.vue [hunk 5]
```diff
@@ -10,9 +10,13 @@ export default {
     handleButton(){
     },
     methods:{
-        deleteObject(id) {
-            this.$delete(this.items, id);
+        removeItem(event) {  
+            console.log(event);
+            this.$emit("remove", );
         }
```
Comment (jan-gajdowicz) [inline src/components/ItemRemover.vue]: Remove, leave only `removeItem() {}`

File: src/components/List.vue [hunk 6]
```diff
@@ -1,7 +1,9 @@
 <template>
   <div>
     <ul>
-      <li v-for="item in items" :key="item.id">{{ item.value }}</li>
+      <li v-for="item in items" :key="item.id">{{ item.value }} 
+        <button v-on:click="handleRemoveButton(item.id)">remove</button>
+      </li>
```
Comment (jan-gajdowicz) [inline src/components/List.vue]:
```
<li v-for="item in items" :key="item.id">
   { item.value }} 
   <button v-on:click="handleRemoveButton(item.id)">remove</button>
</li>
```

**Review by jan-gajdowicz — CHANGES_REQUESTED** [review]
Couple minor things

File: src/App.vue (hunk 1 above)
Comment (JanekJaniak) [reply src/App.vue]: Done

File: src/App.vue (hunk 2 above)
Comment (JanekJaniak) [reply src/App.vue]: done

File: src/components/ItemCreator.vue (hunk 3 above)
Comment (JanekJaniak) [reply src/components/ItemCreator.vue]: done

File: src/components/ItemCreator.vue (hunk 4 above)
Comment (JanekJaniak) [reply src/components/ItemCreator.vue]: ✔️

File: src/components/ItemRemover.vue (hunk 5 above)
Comment (JanekJaniak) [reply src/components/ItemRemover.vue]: ✔️

File: src/components/List.vue (hunk 6 above)
Comment (JanekJaniak) [reply src/components/List.vue]: ✔️

**Review by jan-gajdowicz — APPROVED** [review]
GJ! :bike:

**Pull request merged**